internal/app/machined/pkg/controllers/k8s/control_plane.go

```diff
@@ -167,6 +167,7 @@ func NewControlPlaneControllerManagerController() *ControlPlaneControllerManager
 					ExtraArgs:            cfgProvider.Cluster().ControllerManager().ExtraArgs(),
 					ExtraVolumes:         convertVolumes(cfgProvider.Cluster().ControllerManager().ExtraVolumes()),
 					EnvironmentVariables: cfgProvider.Cluster().ControllerManager().Env(),
+					Resources:            convertResources(cfgProvider.Cluster().APIServer().Resources()),
```

Comment:
should be `ControllerManager().Resources()`? (and similar below?)

Reply:
yes :flushed: 

Also didn't noticed during testing as I just also copy pasted the limits :(

fixed now